### ISSUE TITLE
n:block with helpers doesn't work

### ISSUE THREAD

**Issue by janedbal** [nette/latte#23]
Following code doesn't modify the source at all:

```html
<div n:block="|strip|trim">
     <span>value</span>
</div>
```

But this code works:
```smarty
<div>{block |strip|trim}
     <span>value</span>
{/block}</div>
```

Am I missing something or is that a bug?

**Comment by milo** [comment]
> Attribute macros can not contain modifiers.

**Comment by janedbal** [comment]
@milo So this will not be implemented? I was hoping it will be in 2.4 since @dg added such milestone to this issue.

**Comment by milo** [comment]
It will not. It has only few use cases and only few filters can be used on HTML.
